remoulade/middleware/logging_metadata.py

```diff
@@ -0,0 +1,29 @@
+from ..logging import get_logger
+from .middleware import Middleware
+
+
+class LoggingMetadata(Middleware):
+    def __init__(self):
+        self.logger = get_logger(__name__, type(self))
```

Comment:
I think logging_metadata_getter should be defined at this level too and you will need to use you're function to get the right option.